src/extension/src/ProcessHandler.py

```diff
@@ -95,6 +96,9 @@ def start_daemon(self, seq_no, config_settings, ext_env_handler):
             self.logger.log("New shell process launched successfully. [Process ID (PID)={0}]".format(str(process.pid)))
             did_process_start = self.__check_process_state(process, seq_no)
             return process if did_process_start else None
+
+        # Clear temp folder since core process launch failed
+        ext_env_handler.delete_temp_folder_contents()
```

Comment:
is this needed? Enable action has already cleanup the content before we get there

Reply:
Not needed right now since we only have one use of the tmp folder (i.e. save only one file type from core).

But tmp folder can have more use cases in future. In that case, this could be a fail safe method to ensure handler cleans everything if a core launch fails (if things are introduced to the tmp folder while launching core)